packages/ckeditor5-code-block/tests/codeblockui.js

```diff
@@ -197,6 +197,14 @@ describe( 'CodeBlockUI', () => {
 				expect( button ).to.have.property( 'isToggleable', true );
 			} );
 
+			it( 'arrow has the base properties', () => {
+				const dropdown = editor.ui.componentFactory.create( 'codeBlock' );
+				const buttonArrow = dropdown.buttonView.arrowView;
+
+				expect( buttonArrow ).to.have.property( 'label', 'Insert code block' );
+				expect( buttonArrow ).to.have.property( 'tooltip', true );
```

Comment:
Assertions like this belong in the tests of the `SplitButtonView` (which are missing in this PR). 

`SplitButtonView` binds these properties and if you test that fact once in ckeditor5-ui, you don't have repeat yourself over and over again in features that utilize the `SplitButtonView` API (code block, highlight, image, list...).

Since none of the mentioned features (code block, highlight, image, list...) customizes the label/tooltip, they don't need to test it at all. They should only make sure `SplitButtonView` is being used, that's all.